tests/unit/test_api_routing.py

```
@@ -0,0 +1,246 @@
+            self.predict_emotion_patcher.start()
+            self.get_model_status_patcher.start()
+
+            self.addCleanup(self.predict_emotion_patcher.stop)
+            self.addCleanup(self.get_model_status_patcher.stop)
+
+            self.app = app.test_client()
+            self.app.testing = True
+            self.api_available = True
+        except (ImportError, OSError) as e:
+            import warnings
+            warnings.warn(f"Could not import secure_api_server: {e}")
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)